README.md

```diff
@@ -250,17 +267,20 @@ cleanMock();
 
 ## Changelog
 
-#### `2.0.0` - **BREAKING**
+#### `authmosphere 1.0.0` - **BREAKING**
+
+Modified signature of `createAuthCodeRequestUri`, see migration guide for more information.
+
```

Comment:
not missing, but not explicitly visible either - maybe something like a line to mark the version break - it's just cosmetics